conf/core/ConfigOptions.asciidoc

```diff
@@ -1299,6 +1407,13 @@ hoot::GeometryModifierOp.
 The maximum number of parsed tag string values stored in an in-memory cache when reading
 GeoNames data.
 
+=== graph.comparator.max.threads
```

Comment:
You can see other references to this in CompareCmdTest.sh and Testing.conf. Basically, while making changes I produced two separate bug, one of which occurred in a single threaded `GraphComparator` and one in a multithreaded one. So, set the tests to default to single threaded and made increasing max threads optional.